lighthouse-core/test/config/config-test.js

```diff
@@ -94,7 +94,7 @@ describe('Config', () => {
     const configJson = {
       passes: [{
         passName: unlikelyPassName,
-        gatherers: ['viewport'],
+        gatherers: ['meta-elements'],
```

Comment:
I apologize for the number of times I've used `viewport` as a dummy gatherer name :)